WTWSMS/common/landed_titles/old_duchies.txt

```diff
@@ -82,7 +82,8 @@ d_transylvania = {
 	
 	capital = 519 # Feher
 	allow = {
-		always = no
+		controls = 519
+		NOT = { has_landed_title = e_byzantium }
```

Comment:
@loup99 

Why? If they are there, why not use them? I tried to put them in places where they made sense.